tests/auth_tests/test_tokens.py

```diff
@@ -140,3 +140,39 @@ def test_secret_lazy_validation(self):
         msg = 'The SECRET_KEY setting must not be empty.'
         with self.assertRaisesMessage(ImproperlyConfigured, msg):
             default_token_generator.secret
+
+    def test_verify_with_secret_fallbacks(self):
+        user = User.objects.create_user('tokentestuser', 'test2@example.com', 'testpw')
+        p1 = PasswordResetTokenGenerator()
+        p1.secret = 'secret'
+        tk = p1.make_token(user)
+        p2 = PasswordResetTokenGenerator()
+        p2.secret = 'newsecret'
+        p2.secret_fallbacks = ['secret']
+        self.assertTrue(p1.check_token(user, tk))
+        self.assertTrue(p2.check_token(user, tk))
```

Comment:
Use `assertIs(…, True/False)` for testing boolean values, rather than `assertTrue()` and `assertFalse()` as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).
```suggestion
        self.assertIs(p1.check_token(user, tk), True)
        self.assertIs(p2.check_token(user, tk), True)
```